### PR TITLE
Word Wrap Problem

### PR DESCRIPTION
**Pull request by Purvak-10**
## Related Issue
- Word Wrap Problem, issue no. #1387

## Proposed Changes
- Given ‘n’ words of various lengths, and a limit on the number of characters that can be put in one line. Assume each word has length less than the width of line. You have to arrange these given words in line.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [ ] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
- Let take an array A={aaa, bbb, cc, dddd} and length of line=6.

wwp
    
w_w_p

### CODE REVIEW

**Comment by Purvak-10** [comment]
@Mrjoy832 Please check my pr for issue no. #1387.

File: P-Problem Statements & Solution/3-Level 3/Readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Table of content
+- [Word Wrap Problem](#word-wrap-Problem)
+    - [Algorithm](#algorithm)
+    - [Time Complexity](#time-complexity)
+    - [Space Complexity](#space-complexity)
+ 
+
+
```
Comment (Mrjoy832) [inline P-Problem Statements & Solution/3-Level 3/Readme.md:8]: Add your content into level 2 and add index at top of readme

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

File: P-Problem Statements & Solution/3-Level 3/Readme.md (hunk 1 above)
Comment (Purvak-10) [reply P-Problem Statements & Solution/3-Level 3/Readme.md]: @Mrjoy832 Please review, i have done the changes.

File: P-Problem Statements & Solution/2-Level 2/WordWrapProblem.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+int to_print (int a[], int n)
+{
+    int f;
+    if (a[n] == 1)
+        f = 1;
+    else
+        f = to_print(a, a[n]-1) + 1;
+    cout<<"\n line "<<f<<"-> word no. "<<a[n]<<" to "<<n;
+    return f;
+}
```
Comment (Mrjoy832) [inline P-Problem Statements & Solution/2-Level 2/WordWrapProblem.cpp:83]: Add MORE COMMENTS

File: P-Problem Statements & Solution/2-Level 2/Readme.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- You have to arrange these given words in such a way that:
+
+  - Each word is separated by a space character.
+  - You need to put as many as words in every line.
+  - You can not spilt the words and put them in next line.
+  - Extra spaces includes spaces put at the end of every line except the last line.
+  - Example for Extra spaces: Let take an array A={aaa, bbb, cc, dddd} and length of line=6.
+  
+  ![wwp](https://user-images.githubusercontent.com/76229635/167290207-01928ec7-1e65-4f1b-b00c-599bb1b4fca1.jpg)
+### Output
```
Comment (Mrjoy832) [inline P-Problem Statements & Solution/2-Level 2/Readme.md:25]: remove this new file add your content into the existed readme of level-2

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

File: P-Problem Statements & Solution/2-Level 2/Readme.md (hunk 3 above)
Comment (Purvak-10) [reply P-Problem Statements & Solution/2-Level 2/Readme.md]: @Mrjoy832 please review, i have done the changes.

**Comment by Mrjoy832** [comment]
We cannnot accept this much  Commits for single PR..**I am Closing this** create fresh PR with less commits on same issue

**Pull request closed without merging**